phoebe/dependencies/crimpl/common.py

```diff
@@ -270,10 +282,32 @@ def install_conda(self, in_server_directory=False):
         if self.conda_installed:
             return
 
+        python_version = "{}.{}".format(_sys.version_info.major, _sys.version_info.minor)
+
         if in_server_directory:
-            out = self._run_server_cmd("cd {directory}; wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; sh Miniconda3-latest-Linux-x86_64.sh -u -b -p ./crimpl-conda; mkdir ./crimpl-bin; cp ./crimpl-conda/bin/conda ./crimpl-bin/conda".format(directory=self.directory, exportpath=False))
+            out = self._run_server_cmd(
+                "cd {directory}; "
+                "wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; "
+                "sh Miniconda3-latest-Linux-x86_64.sh -u -b -p ./crimpl-conda; "
+                "mkdir ./crimpl-bin; "
+                "cp ./crimpl-conda/bin/conda ./crimpl-bin/conda; "
+                "./crimpl-conda/bin/conda config --set auto_activate_base false; "
+                "./crimpl-conda/bin/conda create -n crimpl python={python_version} --override-channels -c conda-forge -y; ".format(
+                    directory=self.directory, python_version=python_version
```

Comment:
why is this creating an environment here?  Can we move this new logic into `_create_conda_env` or is this basically overriding the base environment?

Reply:
(base) environment doesn't accept python version and always installs the latest. To work around that, I create a (crimpl) environment here that matches the installed python version and needs to be activated from the phoebe side (by way of conda_env parameter).

Reply:
doesn't the phoebe-side create conda environments anyways, through `_create_conda_env`?  I don't see the need to create one here that the user won't even know about since they didn't name it, but maybe I'm missing something